rheem-platforms/rheem-giraph/src/main/java/org/qcri/rheem/giraph/execution/GiraphExecutor.java

```diff
@@ -0,0 +1,103 @@
+package org.qcri.rheem.giraph.execution;
+
+import org.apache.giraph.conf.GiraphConfiguration;
+import org.qcri.rheem.core.api.Configuration;
+import org.qcri.rheem.core.api.Job;
+import org.qcri.rheem.core.optimizer.OptimizationContext;
+import org.qcri.rheem.core.plan.executionplan.ExecutionStage;
+import org.qcri.rheem.core.plan.executionplan.ExecutionTask;
+import org.qcri.rheem.core.platform.*;
+import org.qcri.rheem.core.platform.lineage.ExecutionLineageNode;
+import org.qcri.rheem.core.util.Tuple;
+import org.qcri.rheem.giraph.operators.GiraphExecutionOperator;
+import org.qcri.rheem.giraph.platform.GiraphPlatform;
+
+import java.util.*;
+
+/**
+ * {@link Executor} for the {@link GiraphPlatform}.
+ */
+public class GiraphExecutor extends ExecutorTemplate {
+    private final GiraphPlatform platform;
+
+    private Configuration configuration;
+
+    private Job job;
+
+    private GiraphConfiguration giraphConfiguration;
+
+    public GiraphExecutor(GiraphPlatform platform, Job job) {
+        super(job.getCrossPlatformExecutor());
+        this.job = job;
+        this.platform = platform;
+        this.configuration = job.getConfiguration();
+
+        this.giraphConfiguration = new GiraphConfiguration();
```

Comment:
Is it necessary to somehow configure Giraph according to the job configuration (as we do, for instance, in the `SparkExecutor`)?